{"name": "Проверка ошибки авторизации пользователя при неверном пароле", "status": "passed", "steps": [{"name": "Логин пользователя", "status": "passed", "parameters": [{"name": "data", "value": "{'email': ('bryanscott@example.com', '845#+O+7)i'), 'password': '23322'}"}], "start": 1716901027233, "stop": 1716901027561}, {"name": "Проверка статуса и кода", "status": "passed", "parameters": [{"name": "code", "value": "401"}, {"name": "name", "value": "'success'"}, {"name": "text", "value": "False"}], "start": 1716901027561, "stop": 1716901027561}], "start": 1716901027233, "stop": 1716901027562, "uuid": "42b3622d-cce2-4559-8bac-008d791ed801", "historyId": "1bc09fdfe2830351c17056afbe05115f", "testCaseId": "1bc09fdfe2830351c17056afbe05115f", "fullName": "tests.test_user_login.TestUserLogin#test_user_auth_with_wrong_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_login"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "Sasaraf"}, {"name": "thread", "value": "20240-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_login"}]}